### ISSUE TITLE
[steamshovel] message log not closed automatically (Trac #1518)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1510]
If steamshovel exits normally, it should close the message log automatically.

Migrated from https://code.icecube.wisc.edu/ticket/1518
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:15", 
    "description": "If steamshovel exits normally, it should close the message log automatically.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335655846260", 
    "component": "combo core", 
    "summary": "[steamshovel] message log not closed automatically", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2016-01-20T20:09:10", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160120T21:31:01: hdembinski commented:

> okay, fair proposal. and yet, it was such a tricky thing to decouple the log-viewer lifetime from the steamshovel lifetime in the first place...

**Comment by icecube-trac** [comment]
Trac update at 20160120T21:31:41: hdembinski commented:

> priority "critical"? really?

**Comment by icecube-trac** [comment]
Trac update at 20160120T21:34:06: david.schultz commented:

> It's "critical" in that it will annoy me a lot :)
> 
> You just need to work on your inter-process communication and send the log window a message to close, I guess.  I haven't looked at the setup closely though.

**Comment by icecube-trac** [comment]
Trac update at 20160121T02:14:42: hdembinski commented:

> oh my interprocess communication is really crappy. one process writes a file in /tmp, the other process reads it. :-) yeah, technically it should not be a problem.

**Comment by icecube-trac** [comment]
Trac update at 20160121T02:30:02: 
* hdembinski changed status from "new" to "closed"
* hdembinski commented:

> In changeset 141090:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141090"
> close logviewer automatically if steamshovel does a normal exit, closes #1510
> ```


* hdembinski changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:15: 
* olivas commented:

> Milestone February 2016 deleted


* olivas changed milestone from "February 2016" to ""
